doc/build-windows.md

```diff
@@ -84,6 +85,7 @@ Then build using:
 
     cd depends
     make HOST=i686-w64-mingw32
+    
```

Comment:
Ta, my bad. Although I guess the 32 bit install will have the same problem so maybe add it here as well?